hardhat.config.ts

```diff
@@ -140,6 +140,14 @@ const config: HardhatUserConfig = {
         mnemonic: process.env.MNEMONIC || "",
       },
     },
+    opbnbtestnet: {
+      url: process.env.OPBNB_TESTNET_ARCHIVE_NODE_URL || "https://opbnb-testnet-rpc.bnbchain.org",
```

Comment:
I would rebase on top of `develop`. Following the new pattern, this should be:

```
url: process.env.ARCHIVE_NODE_opbnbtestnet || "https://opbnb-testnet-rpc.bnbchain.org",
```

I would update also the `.env.example` file, and I would add the needed new entries in this file, under the section `etherscan.apiKey` and `etherscan.customChains`, following these instructions to verify the contracts in opbnbscan with hardhat: https://docs.bnbchain.org/opbnb-docs/docs/tutorials/opbnbscan-verify-hardhat-truffle#hardhat

Reply:
I would configure already the `opbnbmainnet` network

Reply:
I would do something similar (rebase + add config for opbnbtestnet and opbnbmainet) also in https://github.com/VenusProtocol/governance-contracts/pull/43